lib/measures/tbd/measure.rb

```diff
@@ -58,6 +58,12 @@ def arguments(model)
     write_tbd_json.setDefaultValue(true)
     args << write_tbd_json
 
+    gen_kiva = OpenStudio::Measure::OSArgument.makeBoolArgument("gen_kiva", true, false)
+    gen_kiva.setDisplayName("Generate Kiva inputs")
+    gen_kiva.setDescription("Generate OSM Kiva settings and objects if model surfaces have foundation boundary conditions")
+    gen_kiva.setDefaultValue(true)
+    args << gen_kiva
+
     return args
```

Comment:
Checkbox : _checked_ by default, but non-intrusive if no floor surfaces have "Foundation" as outside boundary condition.

Reply:
I'd suggest the behavior be that ground surfaces are converted to foundation surfaces rather than asking the user change surfaces to foundation beforehand.  You could just do a loop through all ground surfaces and change to foundation at the beginning of your measure.  Is there really a use case to simulate some surfaces with ground boundary condition and others with foundation?

Reply:
"I'd suggest the behavior be that ground surfaces are converted to foundation surfaces ..." 
> ... sounds like a good _--force kiva_ option.

"Is there really a use case to simulate some surfaces with ground boundary condition and others with foundation?"
> ... dunno, maybe.

"... rather than asking the user change surfaces to foundation beforehand."
> ... we're not asking users to do anything. Users currently _want_ to use Kiva but the _exposed perimeter_ input makes it complicated within the OpenStudio context. This Kiva option makes things valid and easier to edit. Not perfect, but better than what's available now.

Reply:
The option is in place. Haven't fully tested it yet.

Reply:
Both options are now available, and tested.